routes/events.js

```diff
@@ -7,16 +7,25 @@ module.exports = function(app) {
     // Route for creating events
     app.post('/events/create', function(req, res, next) {
 
-        eventServices.createEvent(req.body, function(err, event) {
+    var auth_token = req.get('Authorization');
+    jwt.verify(auth_token, app.get('superSecret'), function(error, userId) {
```

Comment:
you'll probably want to check to see if the `error` object exists and if it does, log it and sendStatus an error.